Autopilot/Common/Interfaces/SPI.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file SPI.h
+ * @author Ian Frosst
+ * @date March 13, 2017
+ * @copyright Waterloo Aerial Robotics Group 2017 \n
+ *   https://raw.githubusercontent.com/UWARG/PICpilot/master/LICENCE 
+ */
+
+#ifndef SPI_H
+#define	SPI_H
+
+#include <stdint.h>
+
+typedef uint8_t byte;
```

Comment:
i can see the byte type being used in plenty of spaces (such as uart, byte queue, etc). Maybe we should create a Types.h file in the common folder for these sorta definitions?